demo/src/store/store.js

```diff
@@ -41,10 +41,43 @@ function reducer(state = {}, action) {
   return newState;
 }
 
-export const store =
-    createStore(
-        reducer,
-        initialState,
-        window.__REDUX_DEVTOOLS_EXTENSION__ &&
-            window.__REDUX_DEVTOOLS_EXTENSION__(),
-    );
+/**
+ * Save a state to localStorage.
+ * @param {{items:ItemStore}} store The new state.
+ */
+const saveState = (store) => {
+  try {
+    const updatedStore = JSON.stringify(store);
+    localStorage.setItem('state', updatedStore);
+  } catch (err) {
+    // for some reason we can't save, just log an error message
```

Comment:
Nit: Comments should be a complete sentence (including capitalization and period).  Applies to a through comments throughout.

Reply:
Fixed